### ISSUE TITLE
about the models

### ISSUE THREAD

**Issue by GaoHuaZhang** [hygxy/ma_densefusion#3]
I'm sorry to disturb you again！Could please tell me that model have to be in ply or PCD format? Is obj format OK? And does the model have to contain RGB information

**Comment by hygxy** [comment]
the obj is also fine and the RGB information of the model is not necessary

**Comment by GaoHuaZhang** [comment]
Thank you for your reply！ I use your code trained a model, but I couldn't use it. I found that the CAD model unit I used was m, but the model unit in Linemodprocessed datasets was mm. I guess this may be the reason. What is the unit of CAD model you use, and is there any way to convert it,thank you!

**Comment by hygxy** [comment]
yes you are right about the reason. I use mm for my CAD model and I use
blender to convert m to mm.
